solace-spring-cloud-starters/solace-spring-cloud-stream-starter/README.adoc

```diff
@@ -743,6 +764,33 @@ Though note that there are few limitations:
 . `concurrency` &gt; 1 is ignored for polled consumers.
 . Setting `provisionDurableQueue` to `false` disables endpoint configuration validation. Meaning that point 1 cannot be validated. In this scenario, it is the developer's responsibility to ensure that point 1 is followed.
 
+== Batch Consumers
```

Comment:
Might be a good idea to document the need to set `content-type: application/octet-stream` when consuming raw TextMessage, BytesMessage, MapMessage, StreamMessage, XMLContentType messages from the wire. 
Without setting the content-type, SCSt defaults to the `ApplicationJsonMessageMarshallingConverter` message converter and the converter fails due to the fact that the Spring message contains a List

Reply:
Added a tip suggesting to set a `contentType` or `useNativeDecoding=true` if the framework fails to convert the batch message.